domains/panel.mc.sono.json

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "getSono",
+    "email": "sono.arbeit@gmail.com"
+  },
+  "record": {
+    "CNAME": "edge.redirect.pizza."
```

Comment:
Invalid CNAME, should not end in "." should just be a hostname. See docs for more information: https://docs.is-a.dev